src/ruby/devcontainer-feature.json

```diff
@@ -21,7 +21,7 @@
     "customizations": {
         "vscode": {
             "extensions": [
-                "rebornix.Ruby"
+                "Shopify.ruby-lsp"
```

Comment:
```suggestion
                "shopify.ruby-lsp"
```

From 
![image](https://github.com/devcontainers/features/assets/24955913/73245c59-f4be-4908-a342-da94b2b57866)

Reply:
Probably, the `id` is case insensitive - but nice to be in sync.